addons/kotsadm/nightly/deployment/kotsadm-deployment.yaml

```diff
@@ -0,0 +1,259 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kotsadm
+  labels:
+    kots.io/kotsadm: "true"
+    kots.io/backup: velero
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: kotsadm
+  template:
+    metadata:
+      labels:
+        app: kotsadm
+        kots.io/kotsadm: "true"
+        kots.io/backup: velero
+      annotations:
+        backup.velero.io/backup-volumes: backup
+        pre.hook.backup.velero.io/command: '["/backup.sh"]'
+        pre.hook.backup.velero.io/timeout: 10m
+    spec:
+      securityContext:
+        runAsUser: 1001
+      serviceAccountName: kotsadm
+      restartPolicy: Always
+      volumes:
+      - name: kubelet-client-cert
+        secret:
+          secretName: kubelet-client-cert
+      - name: kurl-proxy-kotsadm-tls-cert
+        secret:
+          secretName: kotsadm-tls
+      - name: kotsadm-web-scripts
+        configMap:
+          defaultMode: 511 # hex 777
+          name: kotsadm-web-scripts
+      - name: backup
+        emptyDir: {}
+      - name: migrations
+        emptyDir:
+          medium: Memory
+      containers:
+      - image: kotsadm/kotsadm:v0.0.0-nightly
```

Comment:
Should we add `imagePullPolicy: Never` here? to prevent this from getting updated unexpectedly?

That being said, if you're using nightly in production you're going to get some instability anyways...

Reply:
I think `IfNotPresent`, which should be the default for this tag, is ok here.

Reply:
unless you're running multiple nodes, and kotsadm gets rescheduled to a node without the image